docs/tech_specs/screens_by_alert/tech_spec.md

```diff
@@ -80,6 +64,37 @@ System.convert_time_unit(time2 - time1, :native, :second)
 
 </details>
 
+### Timers and TTLs
+
+In addition to the actual structure of the data being stored in the cache, we also need to carefully choose various timings and cache TTLs to properly keep that data up to date.
+
+TTLs in our case are also somewhat tricky, because some of them will be carried out internally by memcached, and others will need to be carried out by us in Elixir code.
+
+**Timers**
+| timer | interval (sec) | Why? |
+| - | - | - |
+| "self-refresh" job-runner GenServer | 70 | Job should give the slowest screen clients (e-ink, 30 sec refresh rate) time to retry after 1 missing request. |
```

Comment:
So this runner executes every 70s (or whatever interval) non-conditionally? Also, is 30s the slowest refresh rate? Can the interval be 35s, then? Does it benefit us to self-refresh more or less frequently?